backend/src/test/java/com/woowacourse/kkogkkog/acceptance/CouponAcceptanceTest.java

```diff
@@ -346,7 +346,8 @@ public class CouponAcceptanceTest extends AcceptanceTest {
     }
 
     static CouponsResponse 쿠폰_생성을_요청하고(String token, Object data) {
-        ExtractableResponse<Response> response = invokePostWithToken("/api/v2/coupons", token, data);
+        ExtractableResponse<Response> response = invokePostWithToken("/api/v2/coupons", token,
+            data);
         return response.as(CouponsResponse.class);
```

Comment:
지금 PR에서 당장 하자는 것은 아니고 `ExtractableResponse<Response>`의 경우 변수 추론을 사용하는 방식으로 리팩토링하는 것은 어떠할까요?

```suggestion
        final var response = invokePostWithToken("/api/v2/coupons", token, data);
        return response.as(CouponsResponse.class);
```

Reply:
var 키워드를 사용하는거 맞나요? 매우 좋습니다?!